<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(118, 118, 118, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Git_Widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>504</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(118, 118, 118, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="title">
     <string>GIT</string>
    </property>
    <widget class="QTabWidget" name="git_widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>611</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(118, 118, 118, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Welcome</string>
      </attribute>
      <widget class="QLabel" name="Label_picture">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>471</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../GUI-Library-using-QT-master/Wordpress-Guides-Subject-600x628-1.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>211</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 25pt &quot;Segoe Script&quot;;

</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;GIT &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;Extention&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Start_WelcomeButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>290</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);
font: 900 italic 16pt &quot;Segoe UI&quot;;


</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>410</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);
font: 900 italic 16pt &quot;Segoe UI&quot;;

</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;Alaa Abdul Wahab&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Start">
      <attribute name="title">
       <string>Start</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>281</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-style:italic; color:#000000;&quot;&gt;GIT EXTENSIONS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cloneputton_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>290</y>
         <width>241</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">


font: 900 italic 16pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Clone  Reposirory</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Remote_Repository">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>330</y>
         <width>241</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 900 italic 16pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string> Remote Repositories </string>
       </property>
      </widget>
      <widget class="QPushButton" name="Open_Repo">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>241</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">

font: 900 italic 16pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Open Repository</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCreateRepo">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>241</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 900 italic 16pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Create New Repository</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ConFigration_git">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>241</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 900 italic 16pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Git Configration</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Configration</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;E-Mail:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userNameLineEdit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>121</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userEmailLineEdit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ConFigGit">
       <property name="geometry">
        <rect>
         <x>194</x>
         <y>233</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 16pt &quot;Segoe UI&quot;;


</string>
       </property>
       <property name="text">
        <string>Config</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Basic Options </string>
      </attribute>
      <widget class="QTextEdit" name="outputTextEdit">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>182</y>
         <width>591</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 8pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>591</width>
         <height>166</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Enter File Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AddFilelineEddit">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddFile">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;


</string>
            </property>
            <property name="text">
             <string>Create File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Enter Dir Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AddDirLineEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddDir">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Create Dir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonCommit">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Commit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditCommitMessage">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEditFilePath">
            <property name="styleSheet">
             <string notr="true">
background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="Reset_commit">
            <property name="styleSheet">
             <string notr="true">font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStatus">
            <property name="styleSheet">
             <string notr="true">

font: 900 italic 14pt &quot;Segoe UI&quot;;

</string>
            </property>
            <property name="text">
             <string> Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LogPutton">
            <property name="styleSheet">
             <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ClearPutton">
            <property name="styleSheet">
             <string notr="true">

font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Branch Options</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>581</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="Create_Branch">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Create Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Enter Branch Name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BranchNameLineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>581</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="DeleteBranch">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Delete Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Enter Branch Name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BranchNameLineEditDelete">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>581</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="CheckOutBranch">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Check out Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Enter Branch Name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BranchNameLineEditCheckOut">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="outputTextEditBranch">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>581</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 8pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>581</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="MergeBranch">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Merge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Enter Branch Name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BranchNameLineEditMerge">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>581</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="ShowAllBranch">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Show All Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ShowCurrentBranch">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Show Current Branch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Remote Options</string>
      </attribute>
      <widget class="QTextEdit" name="outputTextEditRemoteRepo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>591</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 8pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>331</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pullRemoteButton">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Pull from Remote Repository</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushBranchRemoteButton">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Push Branch to Remote Repository</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fetchRemoteButton">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Fetch from Remote Repository</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Push_complete_Repository">
          <property name="styleSheet">
           <string notr="true">
font: 900 italic 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Push Repository</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
